cloud/amazon/ec2_vpc_vpn_gateway.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This Ansible library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = """
+---
+module: ec2_vpc_vpn_gateway
+short_description: create or remove a vpn gateway
+description:
+    - create or remove a vpn gateway
+version_added: "2.1"
+author: Kristian Jones (@klj613)
+options:
+  name:
+    description:
+      - Unique name of the vpn gateway
+    required: true
+  vpc_id:
+    description:
+      - The VPC to attach to this vpn gateway
+  type:
+    description:
+      - The type of the vpn gateway
+    required: true
+  state:
+    description:
+      - The desired state of the vpn gateway
+    required: true
+    choices: [ 'present', 'absent' ]
+  region:
+    description:
+      - The AWS region to use.  Must be specified if ec2_url is not used. If not specified then the value of the EC2_REGION environment variable, if any, is used.
+    required: false
+    default: null
+    aliases: ['aws_region', 'ec2_region']
+  aws_secret_key:
+    description:
+      - AWS secret key. If not set then the value of the AWS_SECRET_KEY environment variable is used.
+    required: false
+    default: None
+    aliases: ['ec2_secret_key', 'secret_key']
+  aws_access_key:
+    description:
+      - AWS access key. If not set then the value of the AWS_ACCESS_KEY environment variable is used.
+    required: false
+    default: None
+    aliases: ['ec2_access_key', 'access_key']
+
```

Comment:
If you add this bit of magic you can remove the `region`, `aws_secret_key` and `aws_access_key` bits from help

Compare http://docs.ansible.com/ansible/ec2_ami_module.html
to the documentation in it's yaml
```
extends_documentation_fragment:
     - aws
     - ec2
```